aiidalab_widgets_base/viewers.py

```diff
@@ -1165,6 +1165,8 @@ def __init__(self, bands, **kwargs):
 
 
 @register_viewer_widget("process.workflow.workchain.WorkChainNode.")
+@register_viewer_widget("process.calculation.calcfunction.CalcFunctionNode.")
+@register_viewer_widget("process.workflow.workfunction.WorkFunctionNode.")
```

Comment:
I think all of this can be replaced with a process viewer:
```suggestion
@register_viewer_widget("process.")
```

Reply:
I tested this and it does not seem to work. Looking at the code, the viewer matches the whole `obj.node_type` string in the current implementation.

https://github.com/aiidalab/aiidalab-widgets-base/blob/770c6410f17822b9a817d031c60a2651413a305a/aiidalab_widgets_base/viewers.py#L73

Reply:
Ok. I wanted to see if there is an easy way to create viewers for the parent classes and then reuse them to view the children. Doesn't seem to be straightforward. So I would keep it as is for the moment as we have more important things to do.